pkg/loop/internal/relayer/pluginprovider/ext/ccip/test/pricegetter_test.go

```diff
@@ -0,0 +1,91 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"sync"
+	"testing"
+
+	"github.com/hashicorp/consul/sdk/freeport"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+
+	ccippb "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ccip"
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/relayer/pluginprovider/ext/ccip"
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccip"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+)
+
+func Test_staticPriceGetter_Evaluate(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	assert.NoError(t, PriceGetter.Evaluate(ctx, PriceGetter))
+
+	botched := PriceGetter
+	botched.config.Addresses = []cciptypes.Address{"wrong token"}
+	assert.Error(t, PriceGetter.Evaluate(ctx, botched))
+}
+
+func TestPriceGetterGRPC(t *testing.T) {
+	t.Parallel()
+	ctx := tests.Context(t)
+	// create a price registry server
+	port := freeport.GetOne(t)
+	addr := fmt.Sprintf("localhost:%d", port)
+	lis, err := net.Listen("tcp", addr)
+	require.NoError(t, err, "failed to listen on port %d", port)
+	t.Cleanup(func() { lis.Close() })
```

Comment:
Is there an error to check here? If not, then can shorten:
```suggestion
	t.Cleanup(lis.Close)
```